src/flavour.rs

```diff
@@ -152,9 +152,9 @@ impl Flavour {
         }
     }
 
-    /// Returns an iterator over the four delicious Catppuccin flavours.
-    pub fn into_iter() -> std::array::IntoIter<Flavour, 4> {
-        [Self::Latte, Self::Frappe, Self::Macchiato, Self::Mocha].into_iter()
+    /// Returns an array of the four delicious Catppuccin flavours.
+    pub const fn all() -> [Self; 4] {
+        [Self::Latte, Self::Frappe, Self::Macchiato, Self::Mocha]
```

Comment:
this would be a breaking change, which i'm not keen to include just yet. could we keep both methods for now? you could have `into_iter()` make use of `all()` like you did in the tests to avoid duplication.

there is a `v2` branch i was working on that simplifies this crate quite a lot, it's stalled at the moment but i do intend to get back to it. i'd be happy to revisit this part of the API again then when i'm already considering a bunch of breaking changes.

Reply:
c603eab should solve this.